CONTRIBUTING.md

```diff
@@ -85,6 +86,24 @@ You may also want to read
 [The Mysterious PHP RFC Process](https://blogs.oracle.com/opal/post/the-mysterious-php-rfc-process-and-how-you-can-change-the-web)
 for additional notes on the best way to approach submitting an RFC.
 
+## Technical resources
+
+There are a number of technical resources on php-src. Unfortunately, they are
+scattered across different websites, and often outdated. Nonetheless, they can
+provide a good starting point for learning about the fundamentals of the code
+base.
+
+* https://www.phpinternalsbook.com/
+* https://www.npopov.com/
+  * [Internal value representation](https://www.npopov.com/2015/05/05/Internal-value-representation-in-PHP-7-part-1.html), [part 2](https://www.npopov.com/2015/06/19/Internal-value-representation-in-PHP-7-part-2.html)
+  * [HashTable implementation](https://www.npopov.com/2014/12/22/PHPs-new-hashtable-implementation.html)
+  * [Zend Virtual Machine](https://www.npopov.com/2017/04/14/PHP-7-Virtual-machine.html)
+  * [How opcache works](https://www.npopov.com/2021/10/13/How-opcache-works.html), [opcache optimizer](https://www.npopov.com/2022/05/22/The-opcache-optimizer.html)
+* https://wiki.php.net/internals
+* https://qa.php.net/phpt_details.php
+* https://phpinternals.net/
+* https://blog.ircmaxell.com/categories/PHP/
```

Comment:
I think most of articles on that page are obsolete (at least partially) and there are other resources overlapping the rest. If I'm not missing something, I'd remove that link.